projects/angular/src/wizard/wizard-title.ts

```diff
@@ -4,12 +4,11 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { Directive } from '@angular/core';
+import { Directive, Input } from '@angular/core';
 
 @Directive({
-  selector: 'clr-wizard-title, clr-wizard-pagetitle',
+  selector: 'clr-wizard-title',
 })
-export class ClrWizardCustomTags {
-  // No behavior
-  // The only purpose is to "declare" the tag in Angular
+export class ClrWizardTitle {
+  @Input('clrHeadingLevel') headingLevel: '1' | '2' | '3' | '4' | '5' | '6' = '1';
```

Comment:
This should support numbers as well. I would add a `heading-level.ts` file with the following:

```typescript
export type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6 | '1' | '2' | '3' | '4' | '5' | '6';
```